packages/node_modules/@webex/internal-plugin-devices/src/devices.js

```diff
@@ -320,7 +328,28 @@ const Devices = WebexPlugin.extend({
 
   // Ampersand method members
 
-  initialize() {} // Sets up the plugin as well as its events.
+  /**
+   * Initializer method for the devices plugin.
+   *
+   * @override
+   * @param {Array<any>} args - An array of items to be mapped as properties.
+   * @returns {void}
+   */
+  initialize(...args) {
+    // Prototype the extended class in order to preserve the parent member.
+    Reflect.apply(WebexPlugin.prototype.initialize, this, args);
+
+    // Declare the feature collection group names.
+    const featureCollectionNames = ['developer', 'entitlement', 'user'];
```

Comment:
Could we make these constants and exported at the root of the plugin?

Reply:
done